source/analytics/reports/jurisdictions/cdc-central/report-sql/SR10-Multi_Year_Line_Graph_of_Disease_Cases/README.md

```diff
@@ -0,0 +1,17 @@
+# SR10: Multi-Year Line Graph of Disease Cases
+
+## Introduction
+
+Standard report SR10 or Multi-Year Line Graph of Disease Cases generates a multi-year line graph of a disease given its selected state and timeframe. 
```

Comment:
nitpick (optional): Consider adding a punctuation after the name of the report.
`Standard report SR10 or Multi-Year Line Graph of Disease Cases : This report generates...`

Reply:
No no it is appreciated! Let me update the content accordingly.